src/Symfony/Component/Console/Application.php

```diff
@@ -272,6 +281,22 @@ public function setCatchExceptions($boolean)
     }
 
     /**
+     * @return bool Whether errors are caught or not during commands execution
+     */
+    public function areErrorsCaught()
+    {
+        return $this->catchErrors;
+    }
+
+    /**
+     * @param bool $boolean Whether to catch errors or not during commands execution
+     */
+    public function setCatchErrors($boolean)
+    {
+        $this->catchErrors = (bool) $boolean;
+    }
```

Comment:
The addition of these 2 methods does not seem related to this PR and should be removed.